unearthed-arcana/2017/20170206.xml

```diff
@@ -223,5 +223,179 @@
+        </sheet>
+        <spellcasting name="Sorcerer" extend="true">
+            <extend>ID_PHB_SPELL_COMPELLED_DUEL</extend>
+            <extend>ID_PHB_SPELL_SEARING_SMITE</extend>
+            <extend>ID_PHB_SPELL_THUNDEROUS_SMITE</extend>
+            <extend>ID_PHB_SPELL_WRATHFUL_SMITE</extend>
+            <extend>ID_PHB_SPELL_BRANDING_SMITE</extend>
+            <extend>ID_PHB_SPELL_MAGIC_WEAPON</extend>
+            <extend>ID_PHB_SPELL_BLINDING_SMITE</extend>
+            <extend>ID_PHB_SPELL_ELEMENTAL_WEAPON</extend>
+            <extend>ID_PHB_SPELL_STAGGERING_SMITE</extend>
+        </spellcasting>
+      </element>
+      <element name="Stone’s	Durability" type="Archetype Feature" source="Unearthed Arcana" id="ID_WOTC_UA_ARCHETYPE_FEATURE_STONE_SORCERER_STONES_DURABILITY">
+        <supports />
+        <description>
+          <p>At	1st	level,	your	connection	to	stone	gives	you	extra	fortitude. Your	hit	point	maximum	increases	by	1,	and	it	increases	by	1	again	whenever	you	gain	a	level	in	this	class.</p>
+          <p class="indent">As	an	action,	you	can	gain	a	base	AC	of	13	+	 your	Constitution	modifier	if	you	aren’t	wearing	armor,	and	your	skin	assumes	a	stony	appearance.	This	effect	lasts	until	you	end	it	as	a	bonus	action,	you	are	incapacitated,	or	you	don	armor	other	than	a	shield.</p>
+        </description>
+        <sheet>
+          <description>
+            Your	hit	point	maximum	increases	by	1,	and	it	increases	by	1	again	whenever	you	gain	a	level	in	this	class. As	an	action,	gain	a base	AC	of	13	+$(Constitution	modifier).	This	effect	lasts	until	you	end	it	as	a	bonus	action,	you	are	incapacitated,	or	you	don	armor	other	than	a	shield.
+          </description>
+        </sheet>
+        <rules>
+          <stat name="HP" value="1" level="1" />
+          <stat name="HP" value="1" level="2" />
+          <stat name="HP" value="1" level="3" />
+          <stat name="HP" value="1" level="4" />
+          <stat name="HP" value="1" level="5" />
+          <stat name="HP" value="1" level="6" />
+          <stat name="HP" value="1" level="7" />
+          <stat name="HP" value="1" level="8" />
+          <stat name="HP" value="1" level="9" />
+          <stat name="HP" value="1" level="10" />
+          <stat name="HP" value="1" level="11" />
+          <stat name="HP" value="1" level="12" />
+          <stat name="HP" value="1" level="13" />
+          <stat name="HP" value="1" level="14" />
+          <stat name="HP" value="1" level="15" />
+          <stat name="HP" value="1" level="16" />
+          <stat name="HP" value="1" level="17" />
+          <stat name="HP" value="1" level="18" />
+          <stat name="HP" value="1" level="19" />
+          <stat name="HP" value="1" level="20" />
+        </rules>
+      </element>
+
+      <element name="Stone Aegis" type="Archetype Feature" source="Unearthed Arcana" id="ID_WOTC_UA_ARCHETYPE_FEATURE_SEA_SORCERER_SHIFTING_FORM">
```

Comment:
This element has the ID of a copy/pasted element.

Reply:
got it